person.rb

```diff
@@ -1,9 +1,12 @@
+require './nameable'
+
 # Represents a person with basic information such as name, age, and permission to use services.
-class Person
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, parent_permission: true, name: 'Unknown')
+    super(name: name)
```

Comment:
- [ ] Good job but unfortunately when I run your app, I got the error message below ⬇️  about the wrong number of arguments in the initialize method 😢 . Please fix it and make your app works as expected and make sure that Trimmer needs to return this `Maximilan ` 👍 .

![decorate](https://github.com/Daniel-Kigozi/school_library/assets/6615394/68df73ce-5e32-4de5-9876-d7bd66d1b161)